### PR TITLE
Adds Black Dog

### PR DESCRIPTION
**Pull request by philnash**

### CODE REVIEW

File: data/black_dog.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
+  "name": "Black Dog",
+  "tagline": "Hoppy Black Wheat Stout",
```
Comment (sammdec) [inline data/black_dog.json]: Would you mind adding a full stop at the end

**Comment by sammdec** [comment]
Apart from that one things its all looks great :+1:

**Comment by philnash** [comment]
Updated. More coming later!

**Comment by sammdec** [comment]
lgtm :+1: Cheers again!

**Pull request merged**